packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -82,21 +79,29 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 					'is-paid': ! suggestion.is_free,
 				} ) }
 			>
-				{ suggestion.is_free ? (
-					__( 'Free' )
-				) : (
-					<>
-						<span className="domain-picker__free-text"> { __( 'Included in plans' ) } </span>
-						<span className="domain-picker__price-is-paid">
+				{ suggestion.is_free
+					? __( 'Free' )
+					: createInterpolateElement(
+							/* translators: An example would be "Included in paid plan, $15/year", where in mobile view it will only be "$15/year". */
+							__( '<price_text>Included in paid plan,</price_text> <price_cost></price_cost>' ),
```

Comment:
This is clever, and I wonder if it would still be better to stick to two separately translated lines to keep it more simple?

```html
<span>Included in paid plan, %s/year</span>
<span>%s/year</span>
```

No strong opinion about this tho.

Reply:
KISSing here. https://github.com/Automattic/wp-calypso/pull/43346/files#diff-7047bc1505ee7edb4912dcd6277d5169R85-R100

Reply:
The challenge is adding the strikethrough effect on the pricing, e.g. "Included in paid plan, ~$10/year~". That's why `createInterpolateElement` had to be used.

And I think the trouble is the comma, if it's not a comma, it can be two separate language strings, because it's not a sentence.

Reply:
Right, I think I'd expect `__( 'Included in paid plan, <strikethrough>%s/year</strikethrough>' )` for desktop and `__( '%s/year' )` for mobile as translator, if that makes sense?

Reply:
This is actually interesting case since we do send the whole sentence for translation because of `price_cost` placeholder (instead of partial sentence), but the sentence gets split with CSS since we hide the beginning and the cost part gets translated separately. So at the end if kinda becomes split sentence. 🙃

Reply:
> Right, I think I'd expect `__( 'Included in paid plan, <strikethrough>%s/year</strikethrough>' )` for desktop and `__( '%s/year' )` for mobile as translator, if that makes sense?

Is html within translation string valid in this case?

Reply:
> This is actually interesting case since we do send the whole sentence for translation because of `price_cost` placeholder (instead of partial sentence), but the sentence gets split with CSS since we hide the beginning and the cost part gets translated separately. So at the end if kinda becomes split sentence. 🙃
> 
> I'd just err on side of cautious and translate full-sentences to avoid any troubles.

The comma makes it complicated. Looking for to a revision to the design of this one.

Reply:
> Is html within translation string valid in this case?

Yep 👍

Reply:
Ah okay then i'll go with the 2 separate full sentence.

Reply:
Okay make it work!
https://github.com/Automattic/wp-calypso/pull/43346/commits/e4ffcd8318538500028db85dcb0826c9468b0cf5